distribution/debian/install.sh

```diff
@@ -59,7 +59,7 @@ app_guid=$(echo "$app_guid" | tr -d ' ')
 app_guid=${app_guid:-media}
 
 echo "This will install [${app^}] to [$bindir] and use [$datadir] for the AppData Directory"
-echo "${app^} will run as the user [$app_uid] and group [$app_guid]. By continuing, you've confirmed that that user and group will have READ and WRITE access to your Media Library and Download Client Completed Download directories"
+echo "${app^} will run as the user [$app_uid] and group [$app_guid]. By continuing, you've confirmed that user and group will have READ and WRITE access to your Media Library and Download Client Completed Download directories"
```

Comment:
while `that that` appears to be a typo it is actually more correct than what it has been replaced with.

Reply:
`that this` ?